src/main/java/org/prebid/server/vast/VastModifier.java

```diff
@@ -86,14 +102,15 @@ private String appendTrackingUrlToVastXml(String vastXml, String vastUrlTracking
             return vastXml;
         }
 
-        final String impressionUrl = "<![CDATA[" + vastUrlTracking + "]]>";
-        final String openTag = "<Impression>";
+        final String impressionTag = "<Impression><![CDATA[" + vastUrlTracking + "]]></Impression>";
+        final String inlineCloseTag = IN_LINE_TAG.replace("<", "</");
```

Comment:
This is a constant actually, there is no need to create a new string for each invocation.